src/test/java/uk/gov/hmcts/cmc/claimstore/processors/MixMapperTest.java

```diff
@@ -0,0 +1,173 @@
+        String outputJson = processor.toJson(output);
+
+        assertThat(json, CoreMatchers.equalTo(outputJson));
+
+    }
+
+    @Test
+    public void shouldProcessOrganisationToCCDJson() throws JSONException {
+        //given
+        Organisation organisation = SampleParty.builder().organisation();
+
+        String json = processor.toJson(organisation);
+
+        Organisation output = processor.fromJson(json, Organisation.class);
+        String outputJson = processor.toJson(output);
+
+        assertThat(json, CoreMatchers.equalTo(outputJson));
+    }
+
+    @Test
+    public void shouldProcessIndividualDetailsToCCDJson() throws JSONException {
+        //given
+        IndividualDetails individual = SampleTheirDetails.builder().individualDetails();
+
+        String json = processor.toJson(individual);
+
+        IndividualDetails output = processor.fromJson(json, IndividualDetails.class);
+        String outputJson = processor.toJson(output);
+
+        assertThat(json, CoreMatchers.equalTo(outputJson));
+    }
+
+    @Test
+    public void shouldProcessSoleTraderDetailsToCCDJson() throws JSONException {
+        //given
+        SoleTraderDetails soleTrader = SampleTheirDetails.builder().soleTraderDetails();
+
+        String json = processor.toJson(soleTrader);
+
+        SoleTraderDetails output = processor.fromJson(json, SoleTraderDetails.class);
+        String outputJson = processor.toJson(output);
+
+        assertThat(json, CoreMatchers.equalTo(outputJson));
+    }
+
+    @Test
+    public void shouldProcessCompanyDetailsToCCDJson() throws JSONException {
+        //given
+        TheirDetails company = SampleTheirDetails.builder().companyDetails();
+
+        String json = processor.toJson(company);
+
+        TheirDetails output = processor.fromJson(json, TheirDetails.class);
+        String outputJson = processor.toJson(output);
+
+        assertThat(json, CoreMatchers.equalTo(outputJson));
+    }
+
+    @Test
+    public void shouldProcessOrganisationDetailsToCCDJson() throws JSONException {
+        //given
+        OrganisationDetails organisation = SampleTheirDetails.builder().organisationDetails();
+
+        String json = processor.toJson(organisation);
+
+        OrganisationDetails output = processor.fromJson(json, OrganisationDetails.class);
+        String outputJson = processor.toJson(output);
+
+        assertThat(json, CoreMatchers.equalTo(outputJson));
+    }
+
+    @Test
+    public void shouldProcessClaimDataToCCDJson() throws JSONException {
+        //given
+        ClaimData claimData = SampleClaimData.builder().build();
+
+        String json = processor.toJson(claimData);
+
+        ClaimData output = processor.fromJson(json, ClaimData.class);
+        String outputJson = processor.toJson(output);
+
+        assertThat(json, CoreMatchers.equalTo(outputJson));
+    }
+
+    @Test
+    public void shouldProcessAmountBreakDownToCCDJson() throws JSONException {
```

Comment:
Please have separate unit test classes, currently all of these are in this one class.